arm/Microsoft.ContainerService/managedClusters/deploy.bicep

```
@@ -279,9 +282,9 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2021-07-01'
         enabled: httpApplicationRoutingEnabled
       }
       omsagent: {
-        enabled: (omsAgentEnabled && (!empty(workspaceId)))
+        enabled: (omsAgentEnabled && (!empty(monitoringWorkspaceId)))
         config: {
-          logAnalyticsWorkspaceResourceID: ((!empty(workspaceId)) ? workspaceId : null)
+          logAnalyticsWorkspaceResourceID: ((!empty(monitoringWorkspaceId)) ? monitoringWorkspaceId : null)
```

```suggestion
          logAnalyticsWorkspaceResourceID: !empty(monitoringWorkspaceId) ? monitoringWorkspaceId : null
```
Unrelated but not-necessary